source/docs/software/can-devices/pneumatic-hub.rst

```diff
@@ -0,0 +1,16 @@
+Pneumatic Hub
+=============
+
+.. image:: /docs/controls-overviews/images/control-system-hardware/pneumatic-hub.png
+    :alt: The Pneumatic Hub (PCH)
+    :width: 400
+
+The Pneumatic Hub (PCH) is a CAN-based device that provides complete control over the compressor and up to 16 solenoids per module. The PCH is integrated into WPILib through a series of classes that make it simple to use.
```

Comment:
We abbreviate the Pneumatic Hub as PH, not PCH.

Reply:
It looks like that wasn't done in Team Update 1. I used PCH for consistency with the manual. I don't see PH used very much in Rev docs (only found 1), so it seems like until the manual is updated, it would be better to use PCH.

Reply:
Actually it was done in TU1.
![image](https://user-images.githubusercontent.com/10224994/149369339-69f16617-1be1-48b1-9152-f89e575851bd.png)